cypress/integration/User Management/BJ_00_InstructorEdit.js

```diff
@@ -0,0 +1,176 @@
+describe("Fills out form with mock data of instructor from our user.json file, view our form to see if the data is inputed properly, edit our form with edited data, and submits our form", () => {
```

Comment:
Describe the goal of the test, not what you're specifically doing for the test. In this case your should be describe("Editing existing instructor account details") 

If a 5 year old can't understand it, then you didn't give it a good enough name.